packages/grid/x-data-grid/src/hooks/features/filter/useGridFilter.tsx

```diff
@@ -180,17 +180,27 @@ export const useGridFilter = (
 
         let newFilterItems: GridFilterItem[];
         const filterItemOnTarget = filterItemsWithValue.find(
-          (item) => item.columnField === targetColumnField,
+          (item) => item.field === targetColumnField,
         );
 
+        const targetColumn = apiRef.current.getColumn(targetColumnField);
+
         if (filterItemOnTarget) {
           newFilterItems = filterItemsWithValue;
         } else if (props.disableMultipleColumnsFiltering) {
-          newFilterItems = [cleanFilterItem({ columnField: targetColumnField }, apiRef)];
+          newFilterItems = [
+            cleanFilterItem(
+              { field: targetColumnField, operator: targetColumn!.filterOperators![0].value! },
```

Comment:
@flaviendelangle Based on this [conversation](https://github.com/mui/mui-x/pull/3671#discussion_r804695140), do you think this approach (same used in `cleanFilterItem`) should be fine for generating `operator` for new filters?

Reply:
> cleanFilterItem can be removed in a follow up.

It's also generating IDs for filter items apart from adding default operator.

https://github.com/mui/mui-x/blob/21553f7d07906ef2d5e553f7437bf3d60ab3ddc6/packages/grid/x-data-grid/src/hooks/features/filter/gridFilterUtils.ts#L43-L45

Should we rename the function and only handle ID generation part?